third_party/Microsoft/Dump/MPEG2Dump.cpp

```diff
@@ -1282,7 +1282,8 @@ HRESULT CMPEG2Dump::WriteNetwork(PBYTE data, DWORD len)
 		temp[--pos] = (TCHAR) digit+L'0';
 	} while (li.QuadPart);
 	sprintf(request, "WRITE %s %d\r\n", temp+pos, m_bRegOptimizeTransfers ? m_bytesLeftInNetworkWrite : len);
-	size_t dataSize = strlen(request);
+// this throws a warning C4267, but send needs int
+	int dataSize = strlen(request);

```

Comment:
You can just put an (int) cast in there to suppress the warning.

Reply:
Done